dashboard/charts.py

```diff
@@ -85,19 +85,18 @@ def create_choropleth_map(locations):
 
 
 @st.cache_data
-def get_albums_sales_line_graph(chosen_album) -> alt.Chart:
+def get_artist_sales_line_graph(chosen_artist) -> alt.Chart:
     """Returns a line graph of sales over time for a chosen album."""
```

Comment:
I think this docstring needs to be updated, judging by the new function name

Reply:
You are right. Good spot thank you